book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["kitzbergerg"]
```

Comment:
Can you link to doc/authors.md here? Otherwise write something generic like "PermaplanT Contributors"